matrix/Quaternion.hpp

```diff
@@ -195,9 +195,10 @@ class Quaternion : public Vector<Type, 4>
         Quaternion &q = *this;
         Vector3<Type> cr = src.cross(dst);
         float dt = src.dot(dst);
```

Comment:
Making dt const is nice from a readability perspective.

Reply:
done in force-push